engine-rest/engine-rest-openapi/src/main/templates/lib/utils.ftl

```diff
@@ -1,3 +1,4 @@
+<!-- Generates a Query String Parameter JSON object -->
```

Comment:
it can be a boolean parameter as well so
 Generates a Query Parameter JSON object ?

Reply:
I meant "Query String" as the part of the URL where additional HTTP Request parameters can be defined. But your suggestion is more correct according to the term defined in the specs here: https://tools.ietf.org/html/rfc3986#section-3